ts/sagas/startup/generateCryptoKeyPair.ts

```diff
@@ -0,0 +1,32 @@
+import { generate, deleteKey, CryptoError } from "io-react-native-crypto";
+import { call } from "typed-redux-saga/macro";
+import {
+  checkPublicKeyExists,
+  isKeyAlreadyGenerated,
+  setKeyAlreadyGenerated
+} from "../../utils/crypto";
+import { mixpanelTrack } from "./../../mixpanel";
+
+const KEY_NAME = "lp-temp-key";
```

Comment:
Why did you use this value? *-temp-*

Reply:
Because we create the key to test the functionality and then we remove it. In a near future refactoring we will also give the key a more valuable name.